pages/documentation/showcase.md

```diff
@@ -175,12 +175,10 @@ If your project is currently using USWDS and you would like to see it included i
 - [USGS Publications Warehouse](https://pubs.er.usgs.gov/)
 - [VA Caseflow](https://github.com/department-of-veterans-affairs/caseflow)
 - [VA Developer Portal](https://developer.va.gov/)
-- [VA Office of Information and Technology](https://www.oit.va.gov/)
```

Comment:
This website has moved to https://digital.va.gov/. After inspecting a few pages site, it looks like only a few items (banner, footer, and a few buttons) are built using USWDS components. Not sure what the threshold is for including it in the showcase.